format_cef/cef.py

```diff
@@ -85,7 +85,7 @@ def sanitise(s, debug_name):
                 s = s.encode('utf-8')
             s = escape(s)
             if max_len is None:
-                if len(s) <= min_len:
+                if len(s) < min_len:
                     raise ValueError(
                         '{}: String longer than {} characters'.format(
```

Comment:
Some min/max muddling seems to be going on around here.
I think the message is wrong.